### PR TITLE
[BOP-744] initial cni documentation

### PR DESCRIPTION
**Pull request by vikramhh**

### CODE REVIEW

**Comment by ry4nz** [comment]
let's wait for https://github.com/Mirantis/mke-docs/pull/36 merge first.  @ogusarenko Please also review this PR.

File: content/docs/reference/cni/cni.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    vxlanVNI: 10000
+  - provider: kuberouter
+    enabled: false
+    deployWithOperator: false
+  - provider: custom
+    enabled: false
+    deployWithOperator: false
+```
+
+Refer to the following table for detail on all of the conifgurable `network` fields:
```
Comment (cdunkelb) [inline content/docs/reference/cni/cni.md:66]: type conifgurable configurable

**Comment by ry4nz** [comment]
@ogusarenko please take a look

File: content/docs/reference/cni/cni.md (hunk 1 above)
Comment (vikramhh) [reply content/docs/reference/cni/cni.md]: changed as suggested

**Review by ry4nz — APPROVED** [review]

**Comment by ry4nz** [comment]
@ogusarenko Please take a look.

**Comment by ogusarenko** [comment]
I am abandoning this PR as long as the content will land with https://github.com/Mirantis/mke-docs/pull/52. Thank you, Vikram!

**Pull request closed without merging**